src/code/z_fbdemo.c

```diff
@@ -40,6 +40,22 @@ Gfx sTransTileSetupDL[] = {
     gsSPEndDisplayList(),
 };
 
+#define SET_VERTEX(vtx, ob0, ob1, ob2, f, tc0, tc1, n0, n1, n2, alpha) \
```

Comment:
```suggestion
#define SET_VERTEX(vtx, x, y, z, s, t, nx, ny, nz, alpha) \
```
for the f / flag argument, just put 0 inside the macro, it's padding